pallets/parachain-staking/src/lib.rs

```diff
@@ -232,13 +232,12 @@ pub mod pallet {
 		}
 		/// Sort top nominators from greatest to least
 		pub fn sort_top_nominators(&mut self) {
-			self.top_nominators
-				.sort_unstable_by(|a, b| b.amount.cmp(&a.amount));
+			self.top_nominators.sort_by(|a, b| b.amount.cmp(&a.amount));
 		}
 		/// Sort bottom nominators from least to greatest
 		pub fn sort_bottom_nominators(&mut self) {
 			self.bottom_nominators
-				.sort_unstable_by(|a, b| a.amount.cmp(&b.amount));
+				.sort_by(|a, b| a.amount.cmp(&b.amount));
```

Comment:
Today we learned that both sort methods are deterministic which means they were not the source of the problem. Do we still want to keep this change? I remember @notlesh had some concerns about memory usage. I prefer to revert this part.

Reply:
The memory issue was because `sort_by` doesn't sort in-place and makes a temporary allocation of 1/2 the original size. However, the contents are just a `(AccountId, Balance)`, so it's still a small allocation. `bottom_nominators` isn't explicitly bounded, but you could derive some implicit bounds, and each nomination requires minimum 5 MOVR which should prevent abuse. So I'm not worried about it.

Also worth mentioning is that a stable sort will help maintain fairness among equal bonds (I hesitate to say that this change is sufficient for fairness, though).

Reply:
Yes, I'll revert it and rename the PR.